landoapi/revisions.py

```diff
@@ -132,6 +134,32 @@ def check_author_planned_changes(*, revision, **kwargs):
     return "The author has indicated they are planning changes to this revision."
 
 
+def check_relman_approval(relman_phid, supported_repos):
+    """Check that Release Managers group approved a revision on a repo
+    with approval required"""
```

Comment:
nit: [PEP 0257](https://www.python.org/dev/peps/pep-0257/) for docstrings, specifically the summary line should always be a single line.